operator/v1/types_ingress.go

```diff
@@ -1453,6 +1453,30 @@ type IngressControllerTuningOptions struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	MaxConnections int32 `json:"maxConnections,omitempty"`
+
+	// reloadInterval defines the minimum interval at which the router is allowed to reload
+	// to accept new changes. Increasing this value can prevent the accumulation of
+	// HAProxy processes, depending on the scenario. Increasing this interval can
+	// also lessen load imbalance on a backend's servers when using the roundrobin
+	// balancing algorithm. Alternatively, decreasing this value may decrease latency
+	// since updates to HAProxy's configuration can take effect more quickly.
+	//
+	// The value must be a time duration value; see <https://pkg.go.dev/time#ParseDuration>.
+	// Currently, the minimum value allowed is 1s, and the maximum allowed value is
+	// 720h (= 30 days). Minimum and maximum allowed values may change in future versions
+	// of OpenShift.
+	//
+	// An empty reloadInterval tells the IngressController to choose the default, which
```

Comment:
What is the difference between zero and nil?

Reply:
I'm not sure if I understand your question. Are you asking what should be the difference b/w giving the field 0 and nil? I think we want to have the reload interval be the default of 5s if no input is given and 1s if the value given is lower than the floor (i.e. 0s).

Reply:
There is no difference between zero and nil. Both will get the default of 5s.